docs/source/user_guide/automl.ipynb

```diff
@@ -645,17 +1598,1393 @@
+       "                  ('AUC', 0.9346246973365617),\n",
+       "                  ('Precision', 0.8333333333333334),\n",
+       "                  ('F1', 0.7692307692307692),\n",
+       "                  ('Balanced Accuracy Binary', 0.8486682808716708),\n",
+       "                  ('Accuracy Binary', 0.9545454545454546),\n",
+       "                  ('# Training', 134),\n",
+       "                  ('# Validation', 66)]),\n",
+       "     'mean_cv_score': 0.6041570221188047,\n",
+       "     'binary_classification_threshold': 0.47437131050216075}],\n",
+       "   'percent_better_than_baseline_all_objectives': {'Log Loss Binary': 84.84244358059615,\n",
+       "    'MCC Binary': inf,\n",
+       "    'Gini': inf,\n",
+       "    'AUC': 29.49455205811139,\n",
+       "    'Precision': 94.44444444444446,\n",
+       "    'F1': 64.61538461538461,\n",
+       "    'Balanced Accuracy Binary': 26.205609362389026,\n",
+       "    'Accuracy Binary': 5.502789084878645},\n",
+       "   'percent_better_than_baseline': 84.84244358059615,\n",
+       "   'validation_score': 0.5936927052183711}},\n",
+       " 'search_order': [0, 1, 2, 3, 4, 5, 6, 7, 8]}"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "automl.results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Parallel AutoML\n",
+    "\n",
+    "By default, all pipelines in an AutoML batch are evaluated in series.  Pipelines can be evaluated in parallel to improve performance during AutoML search.  This is accomplished by a futures style submission and evaluation of pipelines in a batch.  As of this writing, the pipelines use a threaded model for concurrent evaluation.  This is similar to the currently implemented `n_jobs` parameter in the estimators, which uses increased numbers of threads to train and evaluate estimators."
+   ]
+  },
+  {
+   "cell_type": "markdown",
    "metadata": {},
+   "source": [
+    "### Parallelism with Concurrent Futures\n",
+    "\n",
+    "The `EngineBase` class is robust and extensible enough to support futures-like implementations from a variety of libraries.  The `CFEngine` extends the `EngineBase` to use the native Python concurrent.futures library.  The `CFEngine` supports both thread- and process-level parallelism.  The type of parallelism can be chosen using either the `ThreadPoolExecutor` or the `ProcessPoolExecutor`.  If either executor is passed a `max_workers` parameter, it will set the number of processes and threads spawned.  If not, the default number of processes will be equal to the number of processors available and the number of threads set to five times the number of processors available.\n",
+    "\n",
+    "Note: the cell demonstrating process-level parallelism is commented out due to incompatibility with our ReadTheDocs build.  It can be run successfully locally."
```

Comment:
nit: I think it would be easier to understand why the code is commented out if this sentence is right above the code.

Reply:
@freddyaboulton thoughts?

![image](https://user-images.githubusercontent.com/64713315/127237786-c9efebb6-8a97-41a0-9629-a5252df6d290.png)

Reply:
Looks good!